models/model.js

```diff
@@ -46,3 +47,16 @@ exports.sendCommentsByArticleId = (article_id) => {
       return result.rows;
     });
 };
+
+exports.insertCommentsByArticleId = (article_id, author, body) => {
+  const votes = 0;
+  const created_at = new Date();
+
+  const insertCommentQuery = format(
+    "INSERT INTO comments (article_id, author, body, votes, created_at) VALUES (%L) RETURNING *",
+    [article_id, author, body, votes, created_at]
+  );
```

Comment:
A few things to note about this query string 
- you only have to pass article_id, author and body as the other properties are generated by SQL
- Pg format is somewhat overkill in this situation because a parametrized query would do the job. Pg format is more for inserting massive amounts of data in 1 query, but since this is only a few values, we can insert them as `VALUES ($1, $2, $3)`.